tools/c7n_azure/c7n_azure/entry.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017-2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_azure.resources import vm
+from c7n_azure.resources import vnet
+
```

Comment:
(Minor) extra new line

Reply:
I was trying to figure out what the convention was in the project but there were inconsistencies so we should default to https://www.python.org/dev/peps/pep-0008/#blank-lines (two blank lines) - meaning I think in this case this file is correct and query.py is incorrect

Reply:
Fixed query.py

Reply:
yeah generally we're just holding to running flake8 on files for syntax. pylint is a bit more proscriptive, but not sure we're ready there.